VaticanBAV/et/BAVet132.xml

```diff
@@ -0,0 +1,424 @@
+                            <locus from="118r" to="119r"/>
+                            <title type="complete" ref="LIT2509Weddas#Friday"/>
+                          </msItem>
+                          <msItem xml:id="ms_i1.4.6">
+                            <locus from="119r" to="120r"/>
+                            <title type="complete" ref="LIT2509Weddas#Saturday"/>
+                          </msItem>
+                          <msItem xml:id="ms_i1.4.7">
+                            <locus from="120r" to="121r"/>
+                            <title type="complete" ref="LIT2509Weddas#Sunday"/>
+                          </msItem>                  
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i1.5">
+                          <locus from="121r" to="126r"/>
+                          <title type="complete" ref="LIT1113Anqasa"/>
+                          <explicit xml:lang="gez">
+                            በእንተ፡ <choice><corr>ተሠግዎቱ፡</corr><sic>ተሠግወቱ፡</sic></choice> ለወልደ፡ 
+                            <choice><corr>አምላክ፡</corr><sic>አምላኪ፡</sic></choice> ወብኪ፡ ቅሩበነ፡ ኮነ፡ እምድር<supplied reason="omitted">፡</supplied>
+                            ለማኅደር፡ እስከ<supplied reason="omitted">፡</supplied> ዓርያም። በእንተ፡ <choice><corr>ተሠግዎቱ፡</corr><sic>ተግወቱ፡</sic></choice> ለወልደ፡ አምላክ፡ እምኔኪ፨
+                          </explicit>
+                        </msItem>
+                      </msItem>
+                    </msContents>
+                  
+                    <physDesc>
+                      <objectDesc form="Codex">
+                        <supportDesc>
+                          <support>
+                            <material key="parchment">
+                            </material>
+                          </support>
+
+                          <extent>
+                            <measure unit="leaf" n="128">128</measure>
+                            <measure unit="leaf" type="blank">1</measure>
+                            <locus from="1r" to="1v"/>
+                            <dimensions type="outer" unit="mm">
+                              <height>193</height>
+                              <width>125</width>
+                            </dimensions>
+                            </extent>
+                          
+                            <collation>
+                              <note>According to the catalogue, the manuscript consists of 19 quires and one 
+                              protective bifolio. The third, twelfth and penultimate quire are quaternions, the 
+                              fourth quire has 9 folios and the last quire has 5 folios. The other quires are
+                              quinions. This description does not seem to add up to 128 folios.</note>
+                              <signatures>No quire marks.</signatures>
+                              <!--constat fasciculis XIX sine notis, bifolio pro custodia prae-
```

Comment:
the error must be in total number
to get 128 we must have 
1x2 (protective)
3x8 (quires 3, 12, 14) 
1x9 (4)
1x5 (15)
10x10 (1, 2, 5-11, 13)
so a total of XVI / 16 quires (or 15 if the protective bifolium is not counted in)

Reply:
I'm sorry, it still doesn't add up to me. I think I'll leave it commented out

Reply:
sorry too many quinions )) if they do not count the protective sheets then it must be 9x10 (9 quinions) to get 128+2

Reply:
Yes, "constat fasciculis XIV" would be correct to meet the total number of 128 folios of the codex.